cmd/limactl/editflags/editflags.go

```diff
@@ -60,6 +60,10 @@ func registerEdit(cmd *cobra.Command, commentPrefix string) {
 
 	flags.String("set", "", commentPrefix+"modify the template inplace, using yq syntax")
 
+	if runtime.GOOS != "windows" {
+		flags.Bool("start-at-login", false, commentPrefix+"register(--start-at-login=true)/unregister(--start-at-login=false) the systemd/launchd unit file for instance")
```

Comment:
We should print just `systemd` on Linux and just `launchd` on macOS. This message is already too long, and adding words that don't apply to this host is not helpful for unexperienced users.